### DISCUSSION TITLE
Custom validation message not showing

### DISCUSSION THREAD

**Discussion by OskarD** [react-hook-form/react-hook-form#6305] [Q&A]
I'm using react-hook-form 6.8.2 with React 16.13.1 and normal validation works as expected, but not when inside of the "validate" key.

```
const { trigger, formState, watch, reset } = useFormContext({
    mode: 'all',
});
```
--

```
ref={register({
    required: 'This is required',
    minLength: {
        value: 3,
        message: 'Length must be 3 or more',
    },
    validate: {
        always: (value) => value === 1 || 'This should almost always trigger',
    },
    maxLength: {
        value: 5,
        message: 'max 5',
    },
})}
```
required, minLength and maxLength all work, but always doesn't.

I have tried this: `always: (value) => value === 1 || console.log('This should almost always trigger')` which logs the error in the console

and I have tried this: `validate: (value) => value === 1 || 'This should almost always trigger'` which does nothing either

Why isn't the validation message shown in the UI?

**Comment by bluebill1049** [comment]
provide a codesandobx?
